boards/ek-lm4f120xl/include/board.h

```diff
@@ -46,22 +46,26 @@ extern "C" {
 #define LED1_PIN            GPIO_PIN(5, 2)
 #define LED2_PIN            GPIO_PIN(5, 3)
 
-#define LED_PORT            (GPIO_PORTF_DATA_R)
+/**
+ * @brief Port used for `LED0_ON` and similar implementations
```

Comment:
This is `LED0_ON` and not `@ref LED0_ON` because Doxygen could not resolve that.